### PR TITLE
add os template and docker file

### PR DESCRIPTION
**Pull request by maleck13**

### CODE REVIEW

**Comment by maleck13** [comment]
@philbrookes wanna give this a try when you get chance

File: README.md [hunk 1]
```diff
@@ -56,4 +56,24 @@ sysHandler.go
...
 ```bash
 make test-unit 
+```
+
+## build and publish
+
+```
+env GOOS=linux go build .
+
+docker build -t rhmap/negotiator:0.0.1 . ##change build number
```
Comment (philbrookes) [inline README.md]: Looks like the Dockerfile wasn't commited?

**Comment by maleck13** [comment]
fixed lol

**Comment by philbrookes** [comment]
If you rebase on master, the builds will pass now.

**Pull request merged**